src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -635,73 +637,78 @@ private CancellationTokenRegistration RegisterCancellation(CancellationToken can
         // TODO: Remove this overload once https://github.com/dotnet/roslyn/issues/17287 is addressed
         // and the compiler doesn't lift the span temporary from the call site into the async state
         // machine in debug builds.
-        private void ParseStatusLine(ArraySegment<byte> line, HttpResponseMessage response) =>
+        private static void ParseStatusLine(ArraySegment<byte> line, HttpResponseMessage response) =>
```

Comment:
Since you're in this code anyway :), can we remove this overload per comment above?

Reply:
Unfortunately, no, the C# compiler still prevents span locals in an async method.  @VSadov, is there an open issue tracking that I can link to?  The overall issue I linked to above is closed.